src/account/SupportContact.tsx

```diff
@@ -25,6 +28,28 @@ import { currentAccountSelector } from 'src/web3/selectors'
 
 type Props = StackScreenProps<StackParamList, Screens.SupportContact>
 
+async function exportLogs() {
+  try {
+    Logger.showMessage(i18n.t('supportExportLogsStart'))
+    const logsPath = Logger.getReactNativeLogsFilePath()
+
+    // For now we need to export to a world-readable directory on Android
+    // TODO: use the FileProvider approach so we don't need to do this.
+    // See https://developer.android.com/reference/androidx/core/content/FileProvider
+    // and https://github.com/chirag04/react-native-mail/blame/340618e4ef7f21a29d739d4180c2a267a14093d3/android/src/main/java/com/chirag/RNMail/RNMailModule.java#L106
+    if (Platform.OS === 'android') {
+      const publicPath = `${RNFS.ExternalDirectoryPath}/${path.basename(logsPath)}`
+      await RNFS.copyFile(logsPath, publicPath)
+      return publicPath
```

Comment:
Should we do this on android and ios so we have less platform specialization?  All else being equal this _seems_ like it would simplify this implementation and potential future changes (e.g., see comment about log clean up).